src/nn/Enums.js

```diff
@@ -1264,6 +1264,40 @@ export const OperationCode = {
    */
   MAXIMUM: 65,
 
+  /** Parametric Rectified Linear Unit.
```

Comment:
@huningxin This explanation of PRELU is from [ANEURALNETWORKS_PRELU](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0a82e7e7c709ad117a0178203217be30cf), please take a review, thanks

Reply:
Looks good. Thanks.